### PR TITLE
Cleanup scripts

### PR DESCRIPTION
**Pull request by brybacki**
Removed some logic to reuse what is already provided by kubevirtci

Signed-off-by: Bartosz Rybacki <brybacki@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cluster-up/up.sh [hunk 1]
```diff
@@ -9,25 +9,10 @@ fi
...
-KUBEVIRT_STORAGE=rook-ceph-default
-
 up
 
 # check if the environment has a corrupted host
 if [[ $(${KUBEVIRTCI_PATH}kubectl.sh get nodes | grep localhost) != "" ]]; then
     echo "The environment has a corrupted host"
     exit 1
 fi
-
```
Comment (ShellyKa13) [inline cluster-up/up.sh:22]: what is the purpose of this file now? does it do anything?

Comment (brybacki) [reply cluster-up/up.sh:22]: This actually starts the cluster (`up`) and it is part of `kubevirtci`. You can find exactly the same file in kubevirt or cdi. I want to keep the code that is provided by kubevirtci clean without any changes, and do any additional logic and hacks in `hack/` directory. That is why I moved some code from here to `hack/cluster-up.sh`. 

I am not sure I understand everything about kubevirtci, but I think that now I can easily update kubevirtci to newer version without worrying about merge conflicts.

Comment (brybacki) [reply cluster-up/up.sh:22]: So I want keep the whole `cluster-up` directory clean without kubevirt-velero-plugin changes.

**Pull request merged**